esp/templates/program/modules/programprintables/classattend.html

```diff
@@ -29,56 +30,66 @@
   <br />
   <table align="center" class="pretty" cellspacing="0" cellpadding="3px">
     <tr>
+      {% if "student_profile_pronoun_field"|getBooleanTag %}
+      <th colspan="6" align="center">Students</th>
+      {% else %}
       <th colspan="5" align="center">Students</th>
+      {% endif %}
     </tr>
     <tr>
       <th>#</th>
       <th>Here?</th>
       <th>Student ID</th>
       <th>Student Name</th>
+      {% if "student_profile_pronoun_field"|getBooleanTag %}
+      <th>Pronouns</th>
+      {% endif %}
       <th>Grade</th>
     </tr>{% for student in sec.students %}
     <tr>
       <td align="right">{{ forloop.counter }}</td>
       <td><div style="width: 15px; height: 15px; border: solid black 2px;">&nbsp;</div></td>
       <td>{{ student.id }}</td>
       <td>{{ student.name }}</td>
+      {% if "student_profile_pronoun_field"|getBooleanTag %}
+      <td>{{ student.getLastProfile.student_info.pronoun }}</td>
+      {% endif %}
       <td>
           {{ student|getGradeForProg:program.id }}</td>
     </tr>
     {% endfor %}
     <tr>
       <td>&nbsp; </td>
       <td><div style="width: 15px; height: 15px; border: solid black 2px;">&nbsp;</div></td>
-      <th colspan="3">  
+      <th colspan="4">  
```

Comment:
I think this and the other colspan changes should be conditional on the tag value?